### PR TITLE
Shares backend

### PR DESCRIPTION
**Pull request by Triantafullenia-Doumani**
\OCA\ScienceMesh\Controller::RevaController
  Methods:  78.05% (32/41)   Lines:  76.67% (355/463)

### CODE REVIEW

**Review by ishifoev — APPROVED** [review]
Yes, the test is passed.  If your comment code if don't need you can just remove it.

**Comment by Triantafullenia-Doumani** [comment]
I will probably need it for Unshare().

File: lib/Controller/RevaController.php [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 namespace OCA\ScienceMesh\Controller;
 
-use OCA\ScienceMesh\PlainResponse;
 use OCA\ScienceMesh\NextcloudAdapter;
-use OCA\ScienceMesh\ShareProvider\ScienceMeshShareProvider;
```
Comment (br0kk0l1) [inline lib/Controller/RevaController.php:7]: here

**Review by br0kk0l1 — COMMENTED** [review]
I think this reverts https://github.com/pondersource/nc-sciencemesh/commit/8917f1a3692817797ce5ef910b99cea672e2f8a9, which would break ListReceivedShares again... (see comments)
Maybe you can rebase or just manually change it

the RevaController constructor has changed in these, so it'll probably break the tests as well, but if you just add the shareProvider at the end of every constructor in the tests, they should work again...

**Comment by Triantafullenia-Doumani** [comment]
@br0kk0l1 Ok I updated it.

**Review by br0kk0l1 — APPROVED** [review]
looks good, thank you

**Pull request merged**